### PR TITLE
chore: cleanup styles from Nancy

### PR DESCRIPTION
**Pull request by gabyzif**
## Summary

This PR migrates styles from Nancy's style.less file into Aquarium.

### Components updated

Table

These changes were implemented in the [CDP PR](https://git.corp.mparticle.com/mParticle/mPServer/pull/30679), and they can be tested in [the PR environment](https://app-qa306790.qa.corp.mparticle.com/) associated with it.

## Testing Plan

- [x] Was this tested locally? If not, explain why.
- [x] Was it tested in CDP?
- [x] Was it tested in Indicative? Yes, - locally 

## Reference Issue (For mParticle employees only. Ignore if you are an outside contributor)

- Closes https://go.mparticle.com/work/UNI-1138

### CODE REVIEW

File: src/components/navigation/GlobalNavigation/WorkspaceSelector/WorkspaceSelector.tsx [hunk 1]
```diff
@@ -139,7 +139,7 @@ export function WorkspaceSelector(props: IWorkspaceSelectorProps) {
     <Popover
       arrow={false}
       placement="right"
-      overlayInnerStyle={{ padding: PaddingXxs }}
+      overlayInnerStyle={{ padding: PaddingXxs, width: '100%' }}
```
Comment (gabyzif) [inline src/components/navigation/GlobalNavigation/WorkspaceSelector/WorkspaceSelector.tsx:142]: This is for making sure that the singout button width is 100%. Works as a fix for removing

```
.workspaceSelector__signoutButtonContainer {
  width: 100%;
}
```

[file here](https://git.corp.mparticle.com/mParticle/mPServer/pull/30679/files#diff-2791712ee491efbda8fb763ee38cb3a556f7985963e6d175e1c72fd24f06776dL29-L31
)

File: src/components/navigation/GlobalNavigation/WorkspaceSelector/workspace-selector.css [hunk 2]
```diff
@@ -131,4 +131,4 @@
 
 .workspaceSelector__signoutButton {
   width: 100%;
-}
+}
```
Comment (gabyzif) [inline src/components/navigation/GlobalNavigation/WorkspaceSelector/workspace-selector.css:134]: This two styles are also for making sure the Sing Out button  width is 100% in all the platforms.

File: src/components/navigation/GlobalNavigation/global-navigation.css [hunk 3]
```diff
@@ -257,9 +258,10 @@
 }
 
 .globalNavigation__badge {
+  margin-left: 4px;
```
Comment (gabyzif) [inline src/components/navigation/GlobalNavigation/global-navigation.css:261]: This bring [this](https://git.corp.mparticle.com/mParticle/mPServer/pull/30679/files#diff-2791712ee491efbda8fb763ee38cb3a556f7985963e6d175e1c72fd24f06776dL24-L28) styles to Nancy.

```
.globalNavigation__badge {
  padding: 0;
  margin-left: 4px;
}
```

File: src/components/navigation/GlobalNavigation/global-navigation.css [hunk 4]
```diff
@@ -257,9 +258,10 @@
...
   border: solid 1px var(--color-border);
   border-radius: var(--border-radius-sm);
-  padding: 1px var(--padding-content-vertical-sm);
+  padding: 0;
   box-shadow: none !important;
 }
```
Comment (gabyzif) [inline src/components/navigation/GlobalNavigation/global-navigation.css:266]: This bring [this](https://git.corp.mparticle.com/mParticle/mPServer/pull/30679/files#diff-2791712ee491efbda8fb763ee38cb3a556f7985963e6d175e1c72fd24f06776dL24-L28) styles to Nancy.

```
.globalNavigation__badge {
  padding: 0;
  margin-left: 4px;
}
```

File: src/components/feedback/Modal/modal.css [hunk 5]
```diff
@@ -0,0 +1,3 @@
+.ant-modal .ant-modal-title {
+    font-weight: 600
```
Comment (nastyastavitskaya) [inline src/components/feedback/Modal/modal.css:2]: I thought we moved from 600 to 500 🤔

**Review by nastyastavitskaya — COMMENTED** [review]
I'm not sure what change was it, or maybe it's just me, but "Beta" badge in global nav looks bigger. Maybe we can set it to "small" or something, or it's the same as it was?

**Comment by gabyzif** [comment]
> I'm not sure what change was it, or maybe it's just me, but "Beta" badge in global nav looks bigger. Maybe we can set it to "small" or something, or it's the same as it was?

checked both and they have the same size (@nastyastavitskaya )

Screenshot 2024-10-23 at 3 38 37 PM

File: src/components/feedback/Modal/modal.css (hunk 5 above)
Comment (nastyastavitskaya) [reply src/components/feedback/Modal/modal.css:2]: we have this in css `--font-weight-strong: 500;` . can we have some centered variable to store it? I really don't want to end up in hard-coded values all over the code (it's what WWE had in Nancy and it was painful)

**Review by nastyastavitskaya — APPROVED** [review]
lgtm!

**Review by tibuurcio — APPROVED** [review]

**Pull request merged**